services/QuillLMS/app/controllers/api/v1/activity_sessions_controller.rb

```diff
@@ -103,12 +103,21 @@ def destroy
     data = params.delete(:data)&.permit!
     time_tracking = data && data['time_tracking']
     timespent = @activity_session&.timespent || ActivitySession.calculate_timespent(time_tracking)
+    max_4_bit_integer_size = 2147483647
```

Comment:
Probably worth making this a constant at the top of the file since it never changes.